src/models/Order.js

```diff
@@ -62,9 +62,13 @@ const orderSchema = new mongoose.Schema(
     orderItems: [orderItemSchema],
     expiresAt: {
       type: Date,
-      // Set expiresAt to the future by 2 minutes for orders with the "cancelled" status
+      // Set expiresAt only for orders with the "cancelled" status
       default: function () {
-        return new Date(Date.now() + 2 * 60 * 1000); // 2 minutes
+        if (this.orderStatus === 'cancelled') {
+          return new Date(Date.now() + 2 * 60 * 1000); // 2 minutes
+        }
+        // If the status is not "cancelled", expiresAt is not set
+        return undefined;
```

Comment:
I'm not _sure- if this will get automatically set if you're _updating_ an order (it will for sure happen if you create a `cancelled` order). But we can find out once we get to that step and it seems fine to me to keep it as you have it 👍